### PR TITLE
Match All but 2 remaining naked functions 

### PR DESCRIPTION
**Pull request by PokeCodec**
This leaves "CheckBagHasSpace" and "BattleIntroOpponent1SendsOutMonAnimation" left.

### CODE REVIEW

File: gflib/text.c [hunk 1]
```diff
@@ -1738,18 +1737,18 @@ void DecompressGlyphFont2(u16 glyphId, bool32 isJapanese)
...
         glyphs = gFont2JapaneseGlyphs + (0x100 * (glyphId >> 0x3)) + (0x10 * (glyphId & 0x7));
         DecompressGlyphTile(glyphs, gUnknown_03002F90.unk0);
-        DecompressGlyphTile(glyphs + 0x8, gUnknown_03002F90.unk20);    // gUnknown_03002F90 + 0x40
-        DecompressGlyphTile(glyphs + 0x80, gUnknown_03002F90.unk40);    // gUnknown_03002F90 + 0x20
-        DecompressGlyphTile(glyphs + 0x88, gUnknown_03002F90.unk60);    // gUnknown_03002F90 + 0x60
-        gUnknown_03002F90.unk80 = gFont2JapaneseGlyphWidths[glyphId];     // gGlyphWidth
-        gUnknown_03002F90.unk81 = 14;    // gGlyphHeight
+        DecompressGlyphTile(glyphs + 0x8, gUnknown_03002F90.unk20);   // gUnknown_03002F90 + 0x40
+        DecompressGlyphTile(glyphs + 0x80, gUnknown_03002F90.unk40);  // gUnknown_03002F90 + 0x20
+        DecompressGlyphTile(glyphs + 0x88, gUnknown_03002F90.unk60);  // gUnknown_03002F90 + 0x60
```
Comment (PikalaxALT) [inline gflib/text.c:1742]: Comments don't match code

File: src/main.c [hunk 2]
```diff
@@ -136,9 +134,11 @@ void AgbMain()
     {
         ReadKeys();
 
-        if (gSoftResetDisabled == FALSE
+        if (!gSoftResetDisabled
          && (gMain.heldKeysRaw & A_BUTTON)
-         && (gMain.heldKeysRaw & B_START_SELECT) == B_START_SELECT)
+         && (gMain.heldKeysRaw & B_BUTTON)
+         && (gMain.heldKeysRaw & START_BUTTON)
+         && (gMain.heldKeysRaw & SELECT_BUTTON)) //The reset key combo A + B + START + SELECT
```
Comment (PikalaxALT) [inline src/main.c:141]: Use the `JOY_HELD_RAW` macro here

File: src/pokenav_ribbons_2.c [hunk 3]
```diff
@@ -195,19 +195,19 @@ u32 sub_81D04C4(struct PokenavSub13 *structPtr)
         sub_81D0814(structPtr);
         return 1;
     }
-    else if (gMain.newAndRepeatedKeys & DPAD_DOWN && structPtr->field_8->unk2 < structPtr->field_8->unk0 - 1)
+    if (gMain.newAndRepeatedKeys & DPAD_DOWN && structPtr->field_8->unk2 < structPtr->field_8->unk0 - 1)
```
Comment (PikalaxALT) [inline src/pokenav_ribbons_2.c:198]: use the `JOY_REPEAT` macro

File: src/pokenav_ribbons_2.c [hunk 4]
```diff
@@ -195,19 +195,19 @@ u32 sub_81D04C4(struct PokenavSub13 *structPtr)
...
     {
         structPtr->field_8->unk2++;
         structPtr->field_C = 0;
         sub_81D0814(structPtr);
         return 1;
     }
-    else if (gMain.newKeys & A_BUTTON)
+    if (gMain.newKeys & A_BUTTON)
```
Comment (PikalaxALT) [inline src/pokenav_ribbons_2.c:205]: use the `JOY_NEW` macro (others follow)

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

File: gflib/text.c (hunk 1 above)
Comment (PokeCodec) [reply gflib/text.c:1742]: should I get rid of them?

**Comment by PokeCodec** [comment]
I did it, to the tune of replacing every applicable area with these macros!

**Comment by PokeCodec** [comment]
Fixed!

File: gflib/text.c (hunk 1 above)
Comment (PokeCodec) [reply gflib/text.c]: Nvm, I got rid of them

File: src/pokedex.c [hunk 5]
```diff
@@ -3400,17 +3400,17 @@ static void Task_HandleInfoScreenInput(u8 taskId)
         }
         return;
     }
-    if (((gMain.newKeys & DPAD_LEFT)
-     || ((gMain.newKeys & L_BUTTON) && gSaveBlock2Ptr->optionsButtonMode == OPTIONS_BUTTON_MODE_LR))
+    if (((JOY_NEW(DPAD_LEFT))
```
Comment (PikalaxALT) [inline src/pokedex.c:3403]: Excess parentheses?

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

**Comment by PokeCodec** [comment]
Fixed!

**Review by PikalaxALT — APPROVED** [review]

**Pull request merged**